File: Backend/DiarioDelGelato/Src/DiarioDelGelato.Domain/Entities/TeamMember.cs
using DiarioDelGelato.Domain.Common;

namespace DiarioDelGelato.Domain.Entities
{
    public class TeamMember : BaseEntity
    {
        public string FullName { get; set; }

        public string PersonalId { get; set; }

        public long WorkStartHour { get; set; } //INTEGER as Unix Time, the number of seconds since 1970-01-01 00:00:00 UTC.

        public long WorkStopHour { get; set; }

        public long BreakStartHour { get; set; }

        public long BreakStopHour { get; set; }

        public bool HasCredential { get; set; } //software logon credential => User (identity)

        public string? Photo { get; set; }

    }
}

File: Frontend/angular-diariodelgelato/src/bkup/DiarioDelGelatoFrontendAngular/app/features/team/models/team-member.ts
// team member and schedule break interval - used for conodelgiorno registering and time&attendance
export interface TeamMember {
    id?: number;
    fullName: string;
    workStartHour?: Date;
    workStopHour?: Date;
    breakStartHour?: Date;
    breakStopHour?: Date;
    userName?: string;
    isAdmin?: boolean;
    photo?: string;
}
